sylvia/tests/remote.rs

```diff
@@ -0,0 +1,55 @@
+use cosmwasm_std::{Response, StdResult};
+use sylvia::{contract, types::InstantiateCtx};
+
+pub struct SomeContract {}
+
+pub mod some_interface {
+    use cosmwasm_std::StdError;
+    use sylvia::interface;
+
+    #[interface]
+    pub trait SomeInterface {
+        type Error: From<StdError>;
+    }
+
+    impl SomeInterface for super::SomeContract {
+        type Error = StdError;
+    }
+}
+
+#[contract]
+#[messages(some_interface as SomeInterface)]
+impl SomeContract {
+    #[allow(clippy::new_without_default)]
```

Comment:
Naaaaaaaaaaaaaaaaaaaaaaaaaaaaaaah, this is ZST. Don't even implement new on it, you just create it in place.

Reply:
We require the `new` method in both entry_points and multitests generation. It is currently obligatory to implement it on contract.

Reply:
Ok, makes sense - still, remove `{}`, and just derive default as it is oneliner.

Reply:
We could check if there is a new method and otherwise create the contract like unit type.

Reply:
or require the default trait implementation on it

Reply:
No, don't diverge basing on existence of `new()`. This is a bookcase for Default trait